app/views/mods_panel.py

```diff
@@ -1799,6 +1801,17 @@ def recalculate_internal_errors_warnings(self) -> None:
         logger.info(f"Finished recalculating {self.list_type} list errors")
         return total_error_text, total_warning_text, num_errors, num_warnings
 
+    def _has_replacement(self, package_id: str, dep: str, package_ids_set: set[str]) -> bool:
+        # Get a list of mods that can replace this mod
+        replacements = KNOWN_MOD_REPLACEMENTS.get(dep, set())
+        # Return true if any of the above mods (replacements) are in the mod list
+        # If no replacements exist for dep, returns false
+        for replacement in replacements:
+            if replacement in package_ids_set:
+                logger.debug(f"Missing dependency [{dep}] for [{package_id}] replaced with [{replacement}]")
+                return True
+        return False
```

Comment:
Now logging whenever a replacement is selected over a missing dependency. Example logs
```
[DEBUG][2024-06-19 11:18:20][3640][MainThread][mods_panel][_has_replacement][1811] : Missing dependency [brrainz.harmony] for [kathanon.floatsubmenu] replaced with [zetrith.prepatcher]
[DEBUG][2024-06-19 11:18:20][3640][MainThread][mods_panel][_has_replacement][1811] : Missing dependency [brrainz.harmony] for [theeyeofbrows.fluffybreakdowns] replaced with [zetrith.prepatcher]
[DEBUG][2024-06-19 11:18:20][3640][MainThread][mods_panel][_has_replacement][1811] : Missing dependency [brrainz.harmony] for [vanillaexpanded.vfesecurity] replaced with [zetrith.prepatcher]
[DEBUG][2024-06-19 11:18:20][3640][MainThread][mods_panel][_has_replacement][1811] : Missing dependency [brrainz.harmony] for [aoba.fortress.industrial] replaced with [zetrith.prepatcher]
```